File: Programas em C/Ex15.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    float P, A, R;

    printf("Peso: ");
    scanf("%f", &P);

    printf("Altura: ");
    scanf("%f", &A);

    R = P/(A*A);

    if(R<20){
        printf("Abaixo do peso");
    }
    else if(R<25){
        printf("Peso ideal");
    }
    else{
        printf("Acima do peso");
    }
    return 0;
}

File: Programas em Phyton/Ex15.py
p = float(input('Peso: '))
a = float(input('Altura: '))
IMC = p/(a*a)
print(f"Seu IMC é {IMC:.2f}")
if(IMC<20):
    print("Abaixo do peso")
elif(IMC<25 and IMC>=20):
    print("Peso ideal")
else:
    print("Acima do peso")